test/Microsoft.VisualStudio.LanguageServices.Razor.Test/DefaultTagHelperCompletionServiceTest.cs

```diff
@@ -183,13 +216,9 @@ public void GetElementCompletions_AllowedChildrenAreIgnoredWhenAtRoot()
                     .AllowChildTag("div")
                     .Build(),
             };
-            var expectedCompletions = ElementCompletionResult.Create(new Dictionary<string, HashSet<TagHelperDescriptor>>()
-            {
-                ["p"] = new HashSet<TagHelperDescriptor>(),
-                ["em"] = new HashSet<TagHelperDescriptor>(),
-            });
+            var expectedCompletions = ElementCompletionResult.Create(new Dictionary<string, HashSet<TagHelperDescriptor>>());
 
-            var existingCompletions = new[] { "p", "em" };
+            var existingCompletions = Enumerable.Empty<string>();
```

Comment:
Scoped in the test. The existing completions didn't help validate the tests expectations; they ended up crossing over with the new test I added.